src/tests/functional/plugin/gpu/shared_tests_instances/single_layer_tests/experimental_detectron_prior_grid_generator.cpp

```diff
@@ -0,0 +1,105 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <single_layer_tests/experimental_detectron_prior_grid_generator.hpp>
+#include <common_test_utils/data_utils.hpp>
+#include <functional_test_utils/ov_tensor_utils.hpp>
+
+namespace {
+
+const std::initializer_list<ov::test::subgraph::ExperimentalDetectronPriorGridGeneratorTestParam> params {
+    // flatten = true (output tensor is 2D)
+    {
+        {true, 0, 0, 4.0f, 4.0f},
+        ov::test::static_shapes_to_test_representation({{3, 4}, {1, 16, 4, 5}, {1, 3, 100, 200}})
+    },
+    // task #72587
+    //{
+    //    {true, 3, 6, 64.0f, 64.0f},
+    //    ov::test::static_shapes_to_test_representation({{3, 4}, {1, 16, 100, 100}, {1, 3, 100, 200}})
+    //},
+//    {
+//        {true, 0, 0, 4.0f, 4.0f},
+//        {
+//            // priors
+//            {{-1, -1}, {{3, 4}, {3, 4}}},
+//            // feature_map
+//            {{-1, -1, -1, -1}, {{1, 16, 4, 5}, {1, 16, 100, 100}}},
+//            // im_data
+//            {{-1, -1, -1, -1}, {{1, 3, 100, 200}, {1, 3, 100, 200}}}
+//        }
+//    },
+    // flatten = false (output tensor is 4D)
+    {
+        {false, 0, 0, 8.0f, 8.0f},
+        ov::test::static_shapes_to_test_representation({{3, 4}, {1, 16, 3, 7}, {1, 3, 100, 200}})
+    },
+    // task #72587
```

Comment:
is there is copypaste from somewhere? What reason of the commented data?

Reply:
yes, from cpu plugin

Reply:
so we can either uncomment or remove it. right?

Reply:
probably